### DISCUSSION TITLE
how to change the language of dkan from english to another language?

### DISCUSSION THREAD

**Discussion by tridoxx** [GetDKAN/dkan#3862] [Q&A]
hello, I hope everyone is well, I would like to know if there is any way to change the default language in dkan?, or only by changing the drupal language would this work for the entire site? I don't want to break anything on my machine

I look forward to your response, thank you very much

**Comment by dafeder** [comment]
Hi @tridoxx, depends what you mean. The DKAN admin interface works like any other Drupal module, so changing the language should be fine. 

Also if you want to contribute some string translations to DKAN they would be very welcome. It looks from your profile like you're probably talking about Spanish - we have a several Spanish speakers in the DKAN community so we are actually somewhat equipped to review such translation contributions. Unfortunately, for now we will need to add [offline translations directly to the module repo](https://www.drupal.org/community/contributor-guide/reference-information/localize-drupal-org/working-with-offline/po-and-pot-files) but I'd like to get DKAN back on drupal.org soon so that translations could be contributed the right way -- through https://localize.drupal.org/.

Of course the front end is a very different story, let us know if that's what you meant to discuss!

**Reply by tridoxx** [reply]
hi @dafeder, thanks for you reply, look what I am doing is trying to translate, using the multilanguage directly from the translation section, I made the changes manually, but they are not reflected, for example look for the word "Publisher" and it appeared, and in front I put as wrote in spanish, but didn't have any changes, same for the other parts that I found in the translation that had to do with dkan, but these weren't updated.

I did it through drupal, and it didn't work, and when I filled all that, I downloaded it in a .po, and uploaded it using drush locale, but the same thing happened, I changed the order of the language, and the primary language, but still the same, on the other hand for the front part, if I realized that it touches using the react.


maybe it's a bug, something I see is that when a locale-update lists several modules except for dkan, I don't know if you suddenly have information about it?

I look forward to your response thank you very much

![image](https://user-images.githubusercontent.com/22879539/198620375-bfb59397-4f85-402f-bd2a-cae50872bc67.png)

**Comment by tridoxx** [comment]
well, reviewing and investigating, the front section in the datasets and all this, is generated from the data-catalog-app. I attach the url where you can find the variables

https://github.com/GetDKAN/data-catalog-app/blob/master/src/templates/dataset/index.jsx

![image](https://user-images.githubusercontent.com/22879539/198848348-43118d50-aaef-400c-87f3-07a65b029eb0.png)
